lib/ioki/apis/passenger_api.rb

```diff
@@ -113,6 +113,20 @@ class PassengerApi
         base_path:   [API_BASE_PATH],
         model_class: Ioki::Model::Passenger::Station
       ),
+      Endpoints::Create.new(
+        :logpay_customer,
+        base_path:            [API_BASE_PATH],
+        path:                 'logpay/customer',
```

Comment:
Same path thing as in the previous PR :)

Reply:
Changed!

Reply:
That doesn't work out of the box with `Endpoints::Create`. I'll merge it as is and create another PR for the consistency.